client/src/modules/employees/registry/search.modal.js

```diff
@@ -71,15 +71,21 @@ function EmployeeRegistryModalController(ModalInstance, Functions, Grades, bhCon
   // returns the parameters to the parent controller
   function submit(form) {
     if (form.$invalid) { return; }
-    
+
     // push all searchQuery values into the changes array to be applied
     angular.forEach(vm.searchQueries, function (value, key) {
       if (angular.isDefined(value)) {
         changes.post({ key : key, value : value });
       }
     });
+    // let push defaultQueries values into the filter
+    angular.forEach(vm.defaultQueries, function (value, key) {
+      if (angular.isDefined(value)) {
+        changes.post({ key : key, value : value });
+      }
+    });
 
-    var loggedChanges = changes.getAll();
+    const loggedChanges = changes.getAll();
```

Comment:
Unfortunately, we cannot use ES2015+ on the client. :(  It will break out build process.  However, this would be the correct way to do it on the server.

Could you change this `const` back to a `var`?